### DISCUSSION TITLE
EQ2 feedback please!

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#66] [Show and tell]
Before I write more of these, can I get some feedback on my first example? For reference, this is the linear applications section with problems about distance/rate/time, percent mixtures, value mixtures, etc. My general plan is to:

1. get them to write out pieces they know and assign them a variable
2. figure out how some variables may be related (to get rid of a variable)
3. summarize in a table
4. find an equation that relates part of the table together
5. solve and describe in context. 

Here are screenshots of my first example:
<img width="639" alt="Screenshot 2024-02-20 at 5 36 48 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/83430e1f-e0bb-4bd3-ac6c-a614b470891c">
<img width="648" alt="Screenshot 2024-02-20 at 5 36 57 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/a9b5586d-d41d-4a4f-bae5-b29c3d700386">
<img width="647" alt="Screenshot 2024-02-20 at 5 37 19 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/eefe666d-37a7-44eb-83a4-80ae34ca8e61">

**Comment by siwelwerd** [comment]
I like (a).  For (b), I wonder if something more like "Write an equation relating t_E and t_W" would guide them a little more.  For (c), it's not clear to me what the table should look like -- are there multiple time points?  If so, maybe a partially filled out table would help.  Also, "distance" is a little unclear -- maybe "distance from station"?  Because "distance from other train" is also relevant here (and maybe also interesting in the table?)

**Comment by siwelwerd** [comment]
@kathypinzon You can just leave your feedback in this thread.

**Comment by AbbyANoble** [comment]
@kathypinzon  - It's updated now! https://teambasedinquirylearning.github.io/precalculus/2024/EQ2.html

**Comment by kathypinzon** [comment]
Activity 1.2.3 (b) I would prefer the question as "Which of the statements describes how the times of the eastbound and westbound train are related?" - since the statements aren't really all equations.

**Comment by kathypinzon** [comment]
Activity 1.2.3 (c) is it possible to fill in a portion of the table for them and have them complete the rest? I am not sure they will be able to generate what should go in the distance column in particular.

**Reply by AbbyANoble** [reply]
Do you think the the eastbound train row would be helpful, and then ask them to do the westbound row?

**Reply by kathypinzon** [reply]
yeah, that or maybe a one part of each row?  like put in the distance for eastbound and the time for westbound.

**Comment by kathypinzon** [comment]
Activity 1.2.3 (e) - can you add another part before this that gives the equations?
Now plug in the expressions from your table for d_E and d_W.
85t+75t=400, etc...

**Reply by AbbyANoble** [reply]
Good idea! I'll work on that.

**Comment by kathypinzon** [comment]
Activity 1.2.6 the first sentence is awkward. maybe this instead - 
Jalen needs groceries, so decides to ride his bike to the store.  It takes him half an hour to get there.

I would also suggest saying speed instead of rate.

**Reply by AbbyANoble** [reply]
I knew that sentence was awkward but could not make it better! Haha. Thanks for fixing it!

**Comment by kathypinzon** [comment]
Activity 1.2.6 part (b) - Which of the statements describes how the speed (rate) of the bike trip and the car trip are related?

**Comment by kathypinzon** [comment]
1.2.8 are you intending to include an example of a mixture problem? otherwise the remark feels a bit out of place.

**Reply by AbbyANoble** [reply]
Yep. Just had not gotten there yet and wanted to ask if you think I should style/scaffold in the same way as before or make it more fluency like.

**Reply by kathypinzon** [reply]
I would say one scaffloded - maybe not as much as first questions but a little.  Then one more for fluency.

**Reply by AbbyANoble** [reply]
I finished the scaffolding of the mixture problem if you don't mind glancing over it! It's 1.2.9. Pull request is in, so shouldn't be too long before it's updated!

**Reply by siwelwerd** [reply]
[Activity 1.2.9](https://teambasedinquirylearning.github.io/precalculus/2024/EQ2.html#activity-la-candymixture)

**Reply by kathypinzon** [reply]
looks great!
